### PR TITLE
Prokka version bump to 1.13 and fixed locus tag parameter

### PR DESCRIPTION
**Pull request by Slugger70**
to make it compatible with Roary tool

### CODE REVIEW

File: tools/prokka/prokka.xml [hunk 1]
```diff
@@ -1,14 +1,11 @@
-<tool id="prokka" name="Prokka" version="1.12.0">
-    <description>prokaryotic genome annotation</description>
+<tool id="prokka" name="Prokka" version="1.13.galaxy0">
```
Comment (bgruening) [inline tools/prokka/prokka.xml]: I'm just not sure about the version here. Did we agreed on something on `galaxy` already? If not leave the number schema fro before until we have settled on the new schema.

Comment (Slugger70) [reply tools/prokka/prokka.xml]: It's already been used in a couple of other tools. Dave B did it for the new snpeff

Comment (Slugger70) [reply tools/prokka/prokka.xml]: But I'm happy to change it back too.

Comment (erasche) [reply tools/prokka/prokka.xml]: If you're doing this it should probably be `+galaxy` instead.

**Comment by davebx** [comment]
Thanks @Slugger70

**Pull request merged**